smurfs/src/actions/index.js

```diff
@@ -13,3 +15,57 @@
    U - updateSmurf
    D - deleteSmurf
 */
+
+export function getSmurfs(url) {
+
+  console.log('getSmurfs action creator has been triggered. ');
+  return (dispatch) => {
+    dispatch({type : "LOAD"});
+    axios.get(url)
+         .then( res => {
+           console.log("API get request succeeded!: ", res.data);
+           dispatch({type: "SUCCESS", payload: res.data})
+         })
+         .catch( err => {
+           console.log(err);
+         })
+  }
+}
+
+export function addSmurf(smurf) {
+  console.log('addSmurfs action creator has been triggered.');
+  return (dispatch) => {
+    dispatch({type: "LOAD"});
+    axios.post('http://localhost:3333/smurfs',smurf)
+         .then( res => {
```

Comment:
Urls are also commonly made into a constant and imported where you call the api so if it ever changes you only need to change it in one place